src/Command/BuildDocumentation.php

```diff
@@ -33,32 +33,55 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $versionsArray = [];
 
+        foreach ($input->getArgument('branches') as $branch) {
+            $this->getSubTree($branch, $output, './source_docs/');
+            $versionsArray[$branch] = $foldername;
+        }
+
+        $dumper = new Dumper();
+
+        // Write the passed in versions to the yml file
+        $path = getcwd() . '/app/' . $this->versions;
```

Comment:
This is a `Command` class so it "creates" a "binary", or more explicitly something executable